src/amber/cmd/templates/scaffold/src/views/{{name}}/_form.ecr.ecr

```diff
@@ -23,7 +23,7 @@
 <% when "boolean" -%>
     <div class="checkbox">
       <%="<"%>%= label(<%=":#{field.name}"%>) -%>
-      <%="<"%>%= check_box(<%=":#{field.name}, checked=#{@name}.#{field.name}.to_s == \"1\""%>) -%>
+      <%="<"%>%= check_box(<%=":#{field.name}, checked: #{@name}.#{field.name}.to_s == \"1\""%>) -%>
```

Comment:
Because a bool is stored as 1/0 and I don't think the ORM currently converts that. In the future that should probably be the case. This is still way cleaner than it used to be though.